packages/uniforms-material/src/ErrorField.tsx

```diff
@@ -19,12 +20,15 @@ function Error({
   variant,
   ...props
 }: ErrorFieldProps) {
+  const theme = useTheme();
+  const themeProps = theme.props?.MuiFormControl;
+
   return !error ? null : (
     <FormControl
       error={!!error}
-      fullWidth={!!fullWidth}
-      margin={margin === 'dense' ? margin : undefined}
-      variant={variant}
+      fullWidth={themeProps?.fullWidth ?? !!fullWidth}
+      margin={themeProps?.margin ?? margin}
+      variant={themeProps?.variant ?? variant}
```

Comment:
To match other fields this should do something like this:
```suggestion
      fullWidth={fullWidth ?? themeProps?.fullWidth ?? true}
      margin={margin ?? themeProps?.margin ?? 'dense'}
      variant={variant}
```

Meaning that you can still override the theme with props.

Reply:
We have to remember to remove the default values though (unless we want to set them to `null` to indicate that we want to take the value from theme but I guess this is weird).